### PR TITLE
[WIP] Add BukkitAPI - SpongeAPI comparison to the Docs

### PR DESCRIPTION
**Pull request by Tzky**
This PR intends to provide a list for an easy comparision of BukkitAPI and SpongeAPI. This should make the lifes of several devs easier when switching from Bukkit to Sponge. This list contains the BukkitAPI as well as the additions made for 1.8 by the Spigot Team.

**Please note that this list is still incomplete and WIP**

All credit for the list goes to @kashike! Thanks for providing it!

### CODE REVIEW

**Comment by Spongy** [comment]
Setting up PR reference, please wait...

File: source/plugin/bukkit-sponge-compare.rst [hunk 1]
```diff
@@ -0,0 +1,680 @@
...
+block.ContainerBlock                                   block.tileentity.carrier.TileEntityCarrier
+block.CreatureSpawner                                  block.tileentity.MobSpawner
+block.Dispenser                                        block.tileentity.carrier.Dispenser
+block.DoubleChest
+block.Dropper                                          block.tileentity.carrier.Dropper
+block.Furnace                                          block.tileentity.carrier.Furnace
+block.Hopper                                           block.tileentity.carrier.Hopper
+block.Jukebox                                          block.tileentity.Jukebox
+block.NoteBlock                                        block.tileentity.Note
+block.PistonMoveReaction
```
Comment (kashike) [inline source/plugin/bukkit-sponge-compare.rst]: No replacement currently, we might be missing this. @gabizou

File: source/plugin/bukkit-sponge-compare.rst [hunk 2]
```diff
@@ -0,0 +1,680 @@
...
+configuration.serialization.DelegateDeserialization    (none)
+configuration.serialization.SerializableAs             (none)
+configuration.Configuration                            (none)
+configuration.ConfigurationOptions                     (none)
+configuration.ConfigurationSection                     (none)
+configuration.InvalidConfigurationException            (none)
+configuration.MemoryConfiguration                      (none)
+configuration.MemoryConfigurationOptions               (none)
+configuration.MemorySection                            (none)
+conversations.BooleanPrompt
```
Comment (kashike) [inline source/plugin/bukkit-sponge-compare.rst]: For all in `conversations.`: https://github.com/SpongePowered/SpongeAPI/issues/432

File: source/plugin/bukkit-sponge-compare.rst (hunk 1 above)
Comment (gabizou) [inline source/plugin/bukkit-sponge-compare.rst]: What is it even for?

File: source/plugin/bukkit-sponge-compare.rst [hunk 3]
```diff
@@ -0,0 +1,680 @@
...
+metadata.MetadataValueAdapter
+permissions.Permissible                                service.permission.Subject
+permissions.PermissibleBase
+permissions.Permission
+permissions.PermissionAttachment
+permissions.PermissionAttachmentInfo
+permissions.PermissionDefault
+permissions.PermissionRemovedExecutor
+permissions.ServerOperator
+plugin.java.JavaPlugin
```
Comment (kashike) [inline source/plugin/bukkit-sponge-compare.rst]: No replacement.

File: source/plugin/bukkit-sponge-compare.rst [hunk 4]
```diff
@@ -0,0 +1,680 @@
...
+plugin.java.PluginLoader
```
Comment (kashike) [inline source/plugin/bukkit-sponge-compare.rst]: No replacement.

File: source/plugin/bukkit-sponge-compare.rst [hunk 5]
```diff
@@ -0,0 +1,680 @@
...
+plugin.java.PluginClassLoader
```
Comment (kashike) [inline source/plugin/bukkit-sponge-compare.rst]: No replacement.

File: source/plugin/bukkit-sponge-compare.rst [hunk 6]
```diff
@@ -0,0 +1,680 @@
...
+plugin.messaging.ReservedChannelException
+plugin.messaging.StandardMessenger
+plugin.AuthorNagException
+plugin.EventExecutor
+plugin.IllegalPluginAccessException
+plugin.InvalidDescriptionException
+plugin.InvalidPluginException
+plugin.Plugin
+plugin.PluginAwareness
+plugin.PluginBase
```
Comment (kashike) [inline source/plugin/bukkit-sponge-compare.rst]: No replacement.

File: source/plugin/bukkit-sponge-compare.rst [hunk 7]
```diff
@@ -0,0 +1,680 @@
...
+plugin.PluginDescriptionFile
+plugin.PluginLoader
+plugin.PluginLoadOrder
+plugin.PluginLogger
```
Comment (kashike) [inline source/plugin/bukkit-sponge-compare.rst]: slf4j

File: source/plugin/bukkit-sponge-compare.rst [hunk 8]
```diff
@@ -0,0 +1,680 @@
...
+plugin.PluginManager                                   plugin.PluginManager
+plugin.RegisteredListener
+plugin.RegisteredServiceProvider
+plugin.ServicePriority
+plugin.ServicesManager                                 service.ServiceManager
+plugin.SimplePluginManager
```
Comment (kashike) [inline source/plugin/bukkit-sponge-compare.rst]: No replacement.

File: source/plugin/bukkit-sponge-compare.rst [hunk 9]
```diff
@@ -0,0 +1,680 @@
...
+plugin.SimpleServicesManager                           service.SimpleServiceManager
+plugin.TimedRegisteredListener
```
Comment (kashike) [inline source/plugin/bukkit-sponge-compare.rst]: No replacement.

File: source/plugin/bukkit-sponge-compare.rst [hunk 10]
```diff
@@ -0,0 +1,680 @@
...
+plugin.UnknownDependencyException
```
Comment (kashike) [inline source/plugin/bukkit-sponge-compare.rst]: No replacement.

File: source/plugin/bukkit-sponge-compare.rst [hunk 11]
```diff
@@ -0,0 +1,680 @@
...
+potion.Potion
+potion.PotionBrewer
+potion.PotionEffect                                    effect.potion.PotionEffect
+potion.PotionEffectType                                effect.potion.PotionEffectType
+potion.PotionEffectTypeWrapper
+potion.PotionType
+projectiles.BlockProjectileSource
+projectiles.ProjectileSource
+scheduler.BukkitRunnable
```
Comment (kashike) [inline source/plugin/bukkit-sponge-compare.rst]: No replacement.

File: source/plugin/bukkit-sponge-compare.rst [hunk 12]
```diff
@@ -0,0 +1,680 @@
...
+scheduler.BukkitScheduler
```
Comment (kashike) [inline source/plugin/bukkit-sponge-compare.rst]: `org.spongepowered.api.scheduler.Scheduler`

File: source/plugin/bukkit-sponge-compare.rst [hunk 13]
```diff
@@ -0,0 +1,680 @@
...
+event.Listener
+generator.BlockPopulator
+generator.ChunkGenerator
+help.GenericCommandHelpTopic
+help.HelpMap
+help.HelpTopic
+help.HelpTopicComparator
+help.HelpTopicFactory
+help.IndexHelpTopic
+inventory.meta.BannerMeta
```
Comment (gabizou) [inline source/plugin/bukkit-sponge-compare.rst]: I don't see this in Bukkit API.

File: source/plugin/bukkit-sponge-compare.rst [hunk 14]
```diff
@@ -0,0 +1,680 @@
...
+inventory.meta.BlockStateMeta
```
Comment (gabizou) [inline source/plugin/bukkit-sponge-compare.rst]: Neither this.

File: source/plugin/bukkit-sponge-compare.rst [hunk 15]
```diff
@@ -0,0 +1,680 @@
...
+inventory.meta.BookMeta
```
Comment (gabizou) [inline source/plugin/bukkit-sponge-compare.rst]: Combination of `DisplayNameData`, `AuthorData`, `PageData`, `GenerationData`.

File: source/plugin/bukkit-sponge-compare.rst [hunk 16]
```diff
@@ -0,0 +1,680 @@
...
+inventory.meta.EnchantmentStorageMeta
```
Comment (gabizou) [inline source/plugin/bukkit-sponge-compare.rst]: `StoredEnchantmentData`

File: source/plugin/bukkit-sponge-compare.rst [hunk 17]
```diff
@@ -0,0 +1,680 @@
...
+inventory.meta.FireworkEffectMeta
```
Comment (gabizou) [inline source/plugin/bukkit-sponge-compare.rst]: `data.manipulator.mutable.FireworkEffectData`

File: source/plugin/bukkit-sponge-compare.rst [hunk 18]
```diff
@@ -0,0 +1,680 @@
...
+inventory.meta.FireworkMeta
```
Comment (gabizou) [inline source/plugin/bukkit-sponge-compare.rst]: `FireworkRocketData` and `FireworkEffectData`.

File: source/plugin/bukkit-sponge-compare.rst [hunk 19]
```diff
@@ -0,0 +1,680 @@
...
+inventory.meta.ItemMeta
```
Comment (gabizou) [inline source/plugin/bukkit-sponge-compare.rst]: `DisplayNameData`, `LoreData`, and `EnchantmentData`.

File: source/plugin/bukkit-sponge-compare.rst [hunk 20]
```diff
@@ -0,0 +1,680 @@
...
+inventory.meta.LeatherArmorMeta
```
Comment (gabizou) [inline source/plugin/bukkit-sponge-compare.rst]: `ColorData`.

File: source/plugin/bukkit-sponge-compare.rst [hunk 21]
```diff
@@ -0,0 +1,680 @@
...
+inventory.meta.MapMeta
+inventory.meta.PotionMeta
```
Comment (gabizou) [inline source/plugin/bukkit-sponge-compare.rst]: `PotionEffectData`.

File: source/plugin/bukkit-sponge-compare.rst [hunk 22]
```diff
@@ -0,0 +1,680 @@
...
+event.world.WorldSaveEvent                             event.world.SaveWorldEvent
+event.world.WorldUnloadEvent                           event.world.UnloadWorldEvent
+event.Cancellable                                      event.Cancellable
+event.Event                                            event.Event
+event.EventException
+event.EventHandler                                     event.Listener
+event.EventPriority                                    event.Order
+event.HandlerList
+event.Listener
+generator.BlockPopulator
```
Comment (Deamon5550) [inline source/plugin/bukkit-sponge-compare.rst]: Replaced by `Populator`

File: source/plugin/bukkit-sponge-compare.rst [hunk 23]
```diff
@@ -0,0 +1,680 @@
...
+generator.ChunkGenerator
```
Comment (Deamon5550) [inline source/plugin/bukkit-sponge-compare.rst]: Replaced by `WorldGenerator`

This also has two sub-interfaces in bukkit which are:
`ChunkGenerator.BiomeGrid` -> `BiomeArea`
`ChunkGenerator.ChunkData` - > `BlockVolume`

File: source/plugin/bukkit-sponge-compare.rst [hunk 24]
```diff
@@ -0,0 +1,680 @@
...
+inventory.meta.BookMeta
+inventory.meta.EnchantmentStorageMeta
+inventory.meta.FireworkEffectMeta
+inventory.meta.FireworkMeta
+inventory.meta.ItemMeta
+inventory.meta.LeatherArmorMeta
+inventory.meta.MapMeta
+inventory.meta.PotionMeta
+inventory.meta.Repairable
+inventory.meta.SkullMeta
```
Comment (gabizou) [inline source/plugin/bukkit-sponge-compare.rst]: `RepresentedPlayerData`.

File: source/plugin/bukkit-sponge-compare.rst [hunk 25]
```diff
@@ -0,0 +1,680 @@
...
+util.noise.NoiseGenerator
+util.noise.OctaveGenerator
+util.noise.PerlinNoiseGenerator
+util.noise.PerlinOctaveGenerator
+util.noise.SimplexNoiseGenerator
+util.noise.SimplexOctaveGenerator
+util.permissions.BroadcastPermissions
+util.permissions.CommandPermissions
+util.permissions.DefaultPermissions
+util.BlockIterator
```
Comment (Deamon5550) [inline source/plugin/bukkit-sponge-compare.rst]: Replaced by `BlockRay`

File: source/plugin/bukkit-sponge-compare.rst [hunk 26]
```diff
@@ -0,0 +1,680 @@
...
+inventory.meta.EnchantmentStorageMeta
+inventory.meta.FireworkEffectMeta
+inventory.meta.FireworkMeta
+inventory.meta.ItemMeta
+inventory.meta.LeatherArmorMeta
+inventory.meta.MapMeta
+inventory.meta.PotionMeta
+inventory.meta.Repairable
+inventory.meta.SkullMeta
+inventory.AnvilInventory
```
Comment (gabizou) [inline source/plugin/bukkit-sponge-compare.rst]: `TileEntityInventory`

File: source/plugin/bukkit-sponge-compare.rst [hunk 27]
```diff
@@ -0,0 +1,680 @@
...
+inventory.BeaconInventory
```
Comment (gabizou) [inline source/plugin/bukkit-sponge-compare.rst]: `TileEntityInventory`

File: source/plugin/bukkit-sponge-compare.rst [hunk 28]
```diff
@@ -0,0 +1,680 @@
...
+inventory.BrewerInventory
```
Comment (gabizou) [inline source/plugin/bukkit-sponge-compare.rst]: `TileEntityInventory`

File: source/plugin/bukkit-sponge-compare.rst [hunk 29]
```diff
@@ -0,0 +1,680 @@
...
+PortalType
+Rotation
+SandstoneType                                          data.type.SandstoneType
+Server                                                 Server
+SkullType                                              data.type.SkullType
+Sound                                                  effect.sound.SoundType
+Statistic                                              statistic.Statistic
+TravelAgent
+TreeSpecies                                            data.type.TreeType
+TreeType
```
Comment (Deamon5550) [inline source/plugin/bukkit-sponge-compare.rst]: -> `BiomeTreeType`

File: source/plugin/bukkit-sponge-compare.rst [hunk 30]
```diff
@@ -0,0 +1,680 @@
...
+inventory.meta.ItemMeta
+inventory.meta.LeatherArmorMeta
+inventory.meta.MapMeta
+inventory.meta.PotionMeta
+inventory.meta.Repairable
+inventory.meta.SkullMeta
+inventory.AnvilInventory
+inventory.BeaconInventory
+inventory.BrewerInventory
+inventory.CraftingInventory
```
Comment (gabizou) [inline source/plugin/bukkit-sponge-compare.rst]: Depending, I'm going to assume this is a workbench inventory? If so, it should still be a `GridInventory`, correct me if I'm wrong @Mumfrey

File: source/plugin/bukkit-sponge-compare.rst [hunk 31]
```diff
@@ -0,0 +1,680 @@
...
+inventory.DoubleChestInventory
```
Comment (gabizou) [inline source/plugin/bukkit-sponge-compare.rst]: (none)

File: source/plugin/bukkit-sponge-compare.rst [hunk 32]
```diff
@@ -0,0 +1,680 @@
...
+inventory.EnchantingInventory
+inventory.EntityEquipment
+inventory.EquipmentSlot
```
Comment (gabizou) [inline source/plugin/bukkit-sponge-compare.rst]: `EquipmentSlot` more than likely, though the inventory API is drastically different.

File: source/plugin/bukkit-sponge-compare.rst [hunk 33]
```diff
@@ -0,0 +1,680 @@
...
+inventory.FurnaceInventory
```
Comment (gabizou) [inline source/plugin/bukkit-sponge-compare.rst]: `TileEntityInventory` with an `InputSlot`, `FuelSlot` and `OutputSlot`

File: source/plugin/bukkit-sponge-compare.rst [hunk 34]
```diff
@@ -0,0 +1,680 @@
...
+inventory.FurnaceRecipe
```
Comment (gabizou) [inline source/plugin/bukkit-sponge-compare.rst]: We probably need this (open an API issue someone?)

File: source/plugin/bukkit-sponge-compare.rst [hunk 35]
```diff
@@ -0,0 +1,680 @@
...
+inventory.HorseInventory
```
Comment (gabizou) [inline source/plugin/bukkit-sponge-compare.rst]: `CarriedInventory`

File: source/plugin/bukkit-sponge-compare.rst [hunk 36]
```diff
@@ -0,0 +1,680 @@
...
+inventory.Inventory
```
Comment (gabizou) [inline source/plugin/bukkit-sponge-compare.rst]: `Inventory`

File: source/plugin/bukkit-sponge-compare.rst [hunk 37]
```diff
@@ -0,0 +1,680 @@
...
+inventory.InventoryHolder
```
Comment (gabizou) [inline source/plugin/bukkit-sponge-compare.rst]: `Carrier`

File: source/plugin/bukkit-sponge-compare.rst [hunk 38]
```diff
@@ -0,0 +1,680 @@
...
+inventory.InventoryView
```
Comment (gabizou) [inline source/plugin/bukkit-sponge-compare.rst]: /me throws hands in the air

File: source/plugin/bukkit-sponge-compare.rst [hunk 39]
```diff
@@ -0,0 +1,680 @@
...
+inventory.ItemFactory
```
Comment (gabizou) [inline source/plugin/bukkit-sponge-compare.rst]: Lol, Data API.

File: source/plugin/bukkit-sponge-compare.rst [hunk 40]
```diff
@@ -0,0 +1,680 @@
...
+inventory.ItemFlag
```
Comment (gabizou) [inline source/plugin/bukkit-sponge-compare.rst]: As @kashike mentioned, likely something Spigot added in 1.8.x

File: source/plugin/bukkit-sponge-compare.rst [hunk 41]
```diff
@@ -0,0 +1,680 @@
...
+inventory.ItemStack
```
Comment (gabizou) [inline source/plugin/bukkit-sponge-compare.rst]: `ItemStack`.

File: source/plugin/bukkit-sponge-compare.rst (hunk 34 above)
Comment (kashike) [inline source/plugin/bukkit-sponge-compare.rst]: https://github.com/SpongePowered/SpongeAPI/issues/1027

File: source/plugin/bukkit-sponge-compare.rst (hunk 1 above)
Comment (Deamon5550) [inline source/plugin/bukkit-sponge-compare.rst]: Its for the reaction of a block to being pushed or pulled by a piston, either `RESIST`, `BREAK`, or `MOVE`. Could add a property for it.

File: source/plugin/bukkit-sponge-compare.rst [hunk 42]
```diff
@@ -0,0 +1,680 @@
...
+inventory.MerchantInventory
```
Comment (gabizou) [inline source/plugin/bukkit-sponge-compare.rst]: At the least an `Inventory`, but likely will have more specifics when it's fully implemented.

File: source/plugin/bukkit-sponge-compare.rst [hunk 43]
```diff
@@ -0,0 +1,680 @@
...
+entity.Silverfish                                      entity.living.monster.Silverfish
+entity.Skeleton                                        entity.living.monster.Skeleton
+entity.Slime                                           entity.living.monster.Slime
+entity.SmallFireball                                   entity.projectile.explosive.fireball.SmallFireball
+entity.Snowball                                        entity.projectile.Snowball
+entity.Snowman                                         entity.living.golem.SnowGolem
+entity.Spider                                          entity.living.monster.Spider
+entity.Squid                                           entity.living.Squid
+entity.StorageMinecart                                 entity.vehicle.minecart.MinecartChest
+entity.Tameable
```
Comment (Deamon5550) [inline source/plugin/bukkit-sponge-compare.rst]: `TameableData` ?

File: source/plugin/bukkit-sponge-compare.rst [hunk 44]
```diff
@@ -0,0 +1,680 @@
...
+inventory.FurnaceRecipe
+inventory.HorseInventory
+inventory.Inventory
+inventory.InventoryHolder
+inventory.InventoryView
+inventory.ItemFactory
+inventory.ItemFlag
+inventory.ItemStack
+inventory.MerchantInventory
+inventory.PlayerInventory
```
Comment (gabizou) [inline source/plugin/bukkit-sponge-compare.rst]: `HumanInventory`, though this should change to `PlayerInventory` soon (probably in the bleeding branch already in API).

File: source/plugin/bukkit-sponge-compare.rst [hunk 45]
```diff
@@ -0,0 +1,680 @@
...
+event.enchantment.PrepareItemEnchantEvent
+event.entity.CreatureSpawnEvent
+event.entity.CreeperPowerEvent
+event.entity.EntityBreakDoorEvent
+event.entity.EntityChangeBlockEvent
+event.entity.EntityCombustByBlockEvent
+event.entity.EntityCombustByEntityEvent
+event.entity.EntityCombustEvent
+event.entity.EntityCreatePortalEvent
+event.entity.EntityDamageByBlockEvent
```
Comment (Deamon5550) [inline source/plugin/bukkit-sponge-compare.rst]: -> `DamageEntityEvent`

File: source/plugin/bukkit-sponge-compare.rst [hunk 46]
```diff
@@ -0,0 +1,680 @@
...
+event.player.PlayerStatisticIncrementEvent             event.statistic.ChangeStatisticEvent.TargetPlayer
+event.player.PlayerTeleportEvent                       event.entity.DisplaceEntityEvent.Teleport.TargetPlayer
+event.player.PlayerToggleFlightEvent
+event.player.PlayerToggleSneakEvent
+event.player.PlayerToggleSprintEvent
+event.player.PlayerUnleashEntityEvent
+event.player.PlayerUnregisterChannelEvent
+event.player.PlayerVelocityEvent
+event.server.MapInitializeEvent
+event.server.PluginDisableEvent
```
Comment (Deamon5550) [inline source/plugin/bukkit-sponge-compare.rst]: (none)

File: source/plugin/bukkit-sponge-compare.rst (hunk 43 above)
Comment (kashike) [inline source/plugin/bukkit-sponge-compare.rst]: That's where the values are, yes. No actual replacement for this, though.

**Comment by kashike** [comment]
@Tzky https://gist.github.com/kashike/012cf6835cc10eb93ae7
Still in progress, ping me before updating PR

**Comment by Tzky** [comment]
Updated the PR to separate Spigot 1.8, Spigot and Bukkit 1.7.

**Comment by Tzky** [comment]
Docs staff decided that we'll try host the table elsewhere and not directly on the docs as it will 1. break the docs layout and 2. make downloading/browsing unnecessary harder. We'll probably host this as a public gsheet or similar.

File: source/plugin/bukkit-sponge-compare.rst [hunk 47]
```diff
@@ -0,0 +1,9 @@
+==========================
+Comparing Bukkit to Sponge
+==========================
+
+While the BukkitAPI is notably different to SpongAPI, there are some Events and classes which have an equivalent. This
+page is intended to provide a (probably incomplete) list to help former Bukkit devs out when first trying SpongeAPI.
```
Comment (12AwsomeMan34) [inline source/plugin/bukkit-sponge-compare.rst]: This talks about a page, however you link to a table below it.

**Comment by 12AwsomeMan34** [comment]
Not sure if I like this now, the page looks incredibly bare and empty (look at the preview). Maybe we should point out major differences between Bukkit and Sponge. Possibly related #396

**Comment by Tzky** [comment]
I agree and i'm not happy with the current content either. However @Inscrutable suggested that we move the table elsewhere, as the page was way to lengthy.

If you vor a good idea for additional content, feel free to push commits to this PR. I'll add stuff when i can come up with something :)

File: source/plugin/bukkit-sponge-compare.rst [hunk 48]
```diff
@@ -0,0 +1,9 @@
...
+
+You can view the table here:
+`Sponge to Bukkit comparison Table on GoogleDrive <https://docs.google.com/spreadsheets/d/1phlRmbPHCVhRIaZBi9xMjUuiDQkW-j8rSEV_tQnD47E/edit?usp=sharing>`_
```
Comment (RobertHerhold) [inline source/plugin/bukkit-sponge-compare.rst]: Technically it should probably be 'Google Sheets'

**Comment by Inscrutable** [comment]
Some discussion and comparison between the two API's would help make it look less vacant, but the reason the table exists is that there are few direct analogues. We might also need to make one for each SpongeAPI version too, so that should be noted. 
Having the table away from the docs themselves will make it easier for users to search, and prevent it bloating the docs. Formatting such a large table is already a problem for us. Better ideas always welcome.

File: source/plugin/bukkit-sponge-compare.rst [hunk 49]
```diff
@@ -0,0 +1,11 @@
+==========================
+Comparing Bukkit to Sponge
+==========================
+
```
Comment (Inscrutable) [inline source/plugin/bukkit-sponge-compare.rst]: Typo. SpongeAPI

File: source/plugin/bukkit-sponge-compare.rst [hunk 50]
```diff
@@ -0,0 +1,11 @@
...
+While the BukkitAPI is notably different to SpongAPI, there are some Events and classes which have an equivalent. This
+page is intended to provide a (probably incomplete) list to help former Bukkit devs out when first trying SpongeAPI. The
+table will recieve updates from time to time. If you discovered that something's missing, report it on the
```
Comment (Inscrutable) [inline source/plugin/bukkit-sponge-compare.rst]: Spelling error: receive

File: source/plugin/bukkit-sponge-compare.rst [hunk 51]
```diff
@@ -0,0 +1,11 @@
+==========================
+Comparing Bukkit to Sponge
+==========================
+
+While the BukkitAPI is notably different to SpongAPI, there are some Events and classes which have an equivalent. This
```
Comment (RobertHerhold) [inline source/plugin/bukkit-sponge-compare.rst]: Typo on 'SpongAPI'

**Pull request merged**